Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[workspace]
+members = ["typegen", "description"]
+resolver = "2"
+
+
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+version = "0.0.1"
+rust-version = "1.70.0"
+license = "Apache-2.0 OR GPL-3.0"
+repository = "https://github.com/paritytech/scale-typegen"
+homepage = "https://www.parity.io/"
+
+
+[workspace.dependencies]
+
+scale-typegen-description = { version = "0.0.1", path = "description" }
+scale-typegen = { version = "0.0.1", path = "typegen" }
+
+# external dependencies
+parity-scale-codec = { version = "3.6.5", features = ["derive"] }
+proc-macro2 = "1.0.69"
+quote = "1.0.33"
+scale-bits = "0.4.0"
+scale-info = { version = "2.10.0", features = [
+    "derive",
```

Comment:
nit; Maybe single line here to mirror subxt, or have the other features >= 2 on a separate line for consistency

Reply:
Yeahh I'd stick with single lines in the toml files myself too